src/styles/blocks/card.css

```diff
@@ -0,0 +1,107 @@
+body {
```

Comment:
Move the body styles to global style.css or separate general.css file, as they are not related to the card block.